bundles/org.openhab.binding.herzborg/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="herzborg"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="serial_bus">
+		<label>Herzborg Serial Bus</label>
+		<description>RS485 bus</description>
+		<config-description>
+			<parameter name="port" type="text" required="true">
+				<label>Serial Port</label>
+				<context>serial-port</context>
+				<description>Serial port to use, for example /dev/ttyS0 or COM1</description>
+				<default>/dev/ttyS0</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="curtain">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serial_bus"/>
+		</supported-bridge-type-refs>
+		<label>Herzborg Curtain Motor</label>
+		<description>Curtain motor</description>
+		<channels>
+			<channel id="position" typeId="position"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="reverse" typeId="reverse"/>
+			<channel id="hand_start" typeId="hand_start"/>
+			<channel id="ext_switch" typeId="ext_switch"/>
+			<channel id="hv_switch" typeId="hv_switch"/>
```

Comment:
See https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Channels should be lowerCamelCase to make all bindings consistent and make life easier for textual users.